lib/sqlite.core.js

```diff
@@ -501,6 +501,16 @@ SQLitePluginTransaction.prototype.addStatement = function(sql, values, success,
       } else if (t === 'boolean') {
         //Convert true -> 1 / false -> 0
         params.push(~~v);
+      } else if (Array.isArray(v)) {
+        s = '(';
+        for (i = 0, len2 = v.length; len2 > 1 && i < len2 - 1; i++) {
```

Comment:
This loop is overly complicated for a simple thing it is trying to do. I would suggest to use the following

for (i = 0; i < v.length;i++) {
    s += v[i].toString();
    s = (i < v.length - 1) ? s + "," : s;
}